package.json

```
@@ -28,16 +28,16 @@
     "@solana/spl-token": "^0.1.6",
     "@solana/spl-token-registry": "^0.2.203",
     "@solana/spl-token-swap": "^0.1.0",
-    "@solana/wallet-adapter-ant-design": "^0.1.0",
-    "@solana/wallet-adapter-base": "^0.4.1",
-    "@solana/wallet-adapter-react": "^0.6.0",
-    "@solana/wallet-adapter-wallets": "^0.6.0",
-    "@solana/web3.js": "^1.22.0",
+    "@solana/wallet-adapter-ant-design": "^0.4.1",
+    "@solana/wallet-adapter-base": "^0.5.2",
+    "@solana/wallet-adapter-react": "^0.9.1",
+    "@solana/wallet-adapter-wallets": "^0.7.5",
```

Sorry for the churn! Please test with the latest versions.

```suggestion
    "@solana/wallet-adapter-ant-design": "^0.8.1",
    "@solana/wallet-adapter-base": "^0.6.0",
    "@solana/wallet-adapter-react": "^0.12.5",
    "@solana/wallet-adapter-wallets": "^0.10.1",
```